### PR TITLE
Repo Setup

### PR DESCRIPTION
**Pull request by nitro-marky**
Set up the `wasp-videeo-straming-service`, with the correct test data.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# COPY ./config/stream_dev.conf /etc/nginx/rtmp.d/stream_dev.conf
+# EXPOSE 1935 8080
+# CMD ["/usr/sbin/nginx"]
+
+# FROM base as production
+# COPY ./config/stream_prod.conf /etc/nginx/rtmp.d/stream_prod.conf
+# COPY ./scripts/wasp-video-stream-service.sh /bin/ingest-rtmp.sh
+# ENV ENV=production
+# EXPOSE 1935 8080
+# CMD ["sh", "-c", "/bin/printenv > /etc/envars ; /usr/sbin/nginx"]
```
Comment (dblane-digicatapult) [inline Dockerfile:27]: Copy the dockerfile from the wasp-simple-h264-payload-processor

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (nitro-marky) [reply Dockerfile:27]: And make no name changes anywhere?

File: Dockerfile [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-# COPY ./config/stat.xsl /etc/nginx/stat/stat.xsl
+FROM golang:1.17-alpine AS build
 
-# FROM base as development
-# COPY ./config/stream_dev.conf /etc/nginx/rtmp.d/stream_dev.conf
-# EXPOSE 1935 8080
-# CMD ["/usr/sbin/nginx"]
+WORKDIR /wasp-videeo-stream-service
+COPY . .
+RUN CGO_ENABLED=0 go build -o /bin/wasp-videeo-stream-service
```
Comment (dblane-digicatapult) [inline Dockerfile:6]: `video` rather than `videeo`

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (nitro-marky) [reply Dockerfile:6]: I hate this laptop keyboard!

File: Dockerfile [hunk 3]
```diff
@@ -7,4 +7,4 @@ RUN CGO_ENABLED=0 go build -o /bin/wasp-videeo-stream-service
 
 FROM alpine
 COPY --from=build /bin/wasp-videeo-stream-service /bin/wasp-videeo-stream-service
```
Comment (dblane-digicatapult) [inline Dockerfile]: videeo

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
Delete the .DS_Store and add a sane .gitignore

**Comment by nitro-marky** [comment]
I've added a gitignore but the DS Store isn't in my IDE and I can't delete it manually off the the site.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+## Getting Started
+
+### Requirements
+
+### Building
+
+### Testing
+
+### Linting
+
```
Comment (danielcooperxyz) [inline README.md:12]: Is there any commands you can put under these headings now they're in the makefile?

Comment (nitro-marky) [reply README.md:18]: I've added a few but things like make build just produce a binary which Dave has already told me off about!

Comment (nitro-marky) [reply README.md:18]: Added some commands

**Review by dblane-digicatapult — APPROVED** [review]
LGTM.

**Review by darrylmorton-digicatapult — COMMENTED** [review]
I think the only thing we're missing are the ENVs for kafka?

File: Docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      - KAFKA_CFG_LISTENERS=INTERNAL://:9092,EXTERNAL://:9093
+      - KAFKA_CFG_ADVERTISED_LISTENERS=INTERNAL://kafka:9092,EXTERNAL://localhost:9092
+      - KAFKA_INTER_BROKER_LISTENER_NAME=INTERNAL
+    depends_on:
+      - zookeeper
+  kafka-topics:
+    image: "docker.io/bitnami/kafka:2-debian-10"
+    command: /bin/sh -c "
+      sleep 20 &&
+      kafka-topics.sh --bootstrap-server kafka:9092 --create --topic payloads.simpleH264 --partitions 1 --replication-factor 1 &&
```
Comment (dblane-digicatapult) [inline Docker-compose.yml:31]: Remove, we don't need this topic for egress, just the video one.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# wasp-video-stream-service
+
+## Getting Started
+
+### Requirements
+
+### Building
+
```
Comment (danielcooperxyz) [inline README.md:8]: can you add `make build` here please

Comment (danielcooperxyz) [reply README.md:8]: and also `cleanbuild` too

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**